testsuite/features/srv_delete_channel.feature

```diff
@@ -0,0 +1,82 @@
+# COPYRIGHT (c) 2019 SUSE LLC
+# License under the terms of the MIT License.
+
+Feature: Test delete channels with child or clone
```

Comment:
No, the "feature" is a feature of the SUSE Manager software. Not a test.

s/Test  delete/Delete/

Reply:
I just made a commit changing the feature description. Thanks